functions/Set-DbaMaxMemory.ps1

```diff
@@ -57,28 +60,25 @@ function Set-DbaMaxMemory {
     .EXAMPLE
         PS C:\> Get-DbaCmsRegServer -SqlInstance sqlserver | Test-DbaMaxMemory | Where-Object { $_.SqlMaxMB -gt $_.TotalMB } | Set-DbaMaxMemory
 
-        Find all servers in SQL Server Central Management server that have Max SQL memory set to higher than the total memory
+        Find all servers in SQL Server Central Management Server that have Max SQL memory set to higher than the total memory
         of the server (think 2147483647), then pipe those to Set-DbaMaxMemory and use the default recommendation.
 
 #>
     [CmdletBinding(SupportsShouldProcess = $true, ConfirmImpact = 'Medium')]
```

Comment:
Thank you for all the cleanups! When you see `SupportsShouldProcess = $true` or `Mandatory = $true` can you replace it by `SupportsShouldProcess ` or `Mandatory` ? It's cleaner and good syntax for PS 3